vnext/Desktop/WebSocket.h

```diff
@@ -33,9 +32,9 @@ class BaseWebSocket : public IWebSocket
   boost::beast::multi_buffer m_bufferIn;
   std::thread m_contextThread;
 
-  ///
-  // Must be modified exclusively from the context thread.
-  ///
+  /// <remarks>
```

Comment:
><remarks> [](start = 6, length = 9)

For my own education, what is the doc comment style you are using here? The same as for C#? Also, do we currently have a doc-generating processes operating on the sources in the project?

Reply:
We don't have a doc-generating process, to my knowledge.
I'm following C++ documentation conventions from docs.microsoft.com.
https://docs.microsoft.com/en-us/cpp/build/reference/xml-documentation-visual-cpp?view=vs-2019